src/main/main.dev.ts

```diff
@@ -26,15 +27,21 @@ export default class AppUpdater {
 
 let mainWindow: BrowserWindow | null = null;
 
+ipcMain.on('ipc-example', async (event, arg) => {
```

Comment:
While this might be useful for testing we probably don't want this in the default boilerplate.